.github/workflows/release-code-freeze.yml

```diff
@@ -226,3 +226,49 @@ jobs:
             <${{ needs.modify-zip-version.outputs.artifact_url }}|Download zip>
           slack-optional-unfurl_links: false
           slack-optional-unfurl_media: false
+
+  trigger-webhook:
+    name: Trigger Release Webhook
+    runs-on: ubuntu-latest
+    needs: [modify-zip-version, prepare-for-code-freeze]
+    steps:
+      - name: Trigger Code Freeze Webhook
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea #v7.0.1
+        with:
+          script: |
```

Comment:
Do we want to have this task run even when manually starting the workflow?  We could optionally block it off to only run when the scheduled workflow runs.